frontend/kesaseteli/youth/src/pages/additional_info.tsx

```diff
@@ -0,0 +1,83 @@
+import AdditionalInfoForm from 'kesaseteli/youth/components/forms/AdditionalInfoForm';
+import useYouthApplicationStatusQuery from 'kesaseteli/youth/hooks/backend/useYouthApplicationStatusQuery';
+import AdditionalInfoFormData from 'kesaseteli-shared/types/additional-info-form-data';
+import { GetStaticProps, NextPage } from 'next';
+import Head from 'next/head';
+import { useTranslation } from 'next-i18next';
+import React from 'react';
+import { FormProvider, useForm } from 'react-hook-form';
+import Container from 'shared/components/container/Container';
+import { $Notification } from 'shared/components/notification/Notification.sc';
+import PageLoadingSpinner from 'shared/components/pages/PageLoadingSpinner';
+import useRouterQueryParam from 'shared/hooks/useRouterQueryParam';
+import getServerSideTranslations from 'shared/i18n/get-server-side-translations';
+
+const AdditionalInfo: NextPage = () => {
+  const { t } = useTranslation();
+  const { value: applicationId, isRouterLoading } = useRouterQueryParam('id');
+  const applicationStatusQuery = useYouthApplicationStatusQuery(applicationId);
+
+  const methods = useForm<AdditionalInfoFormData>({
+    mode: 'onBlur',
+    reValidateMode: 'onChange',
+    criteriaMode: 'all',
+  });
+
+  const isLoading = isRouterLoading || applicationStatusQuery.isLoading;
+  const isSuccess = applicationId && applicationStatusQuery.isSuccess;
+  const notFound = applicationStatusQuery.isError || !applicationId;
+
+  if (isLoading) {
+    return <PageLoadingSpinner />;
+  }
+
+  return (
+    <Container>
+      <Head>
+        <title>{`${t(`common:appName`)} - ${t(
+          'common:additionalInfo.title'
+        )}`}</title>
+      </Head>
+      <FormProvider {...methods}>
+        {isSuccess &&
+          (() => {
+            switch (applicationStatusQuery.data.status) {
+              case 'additional_information_requested':
```

Comment:
Can these statuses be moved to an enum..?

Reply:
I think it's matter of taste,  `applicationStatusQuery.data.status` is not string type but literal type: https://www.typescriptlang.org/docs/handbook/literal-types.html#string-literal-types

I use IntelliJ and at least it can autocomplete / suggest the literal type same way as enum:
![Screenshot 2022-03-29 at 9 41 06 AM](https://user-images.githubusercontent.com/7648194/160550442-1eba0ba5-80cf-4c2f-95ab-7a19008f9988.png)
Also shows error when typo:
![Screenshot 2022-03-29 at 9 46 48 AM](https://user-images.githubusercontent.com/7648194/160550447-8d4ffcb1-372f-406c-ab4a-2b61e79688ca.png)